conda/core/solve.py

```diff
@@ -48,21 +48,9 @@ def _get_solver_class(key=None):
 
     See ``context.solver`` for more details.
     """
-    solvers = context.plugin_manager.get_registered_plugins("solvers")
-    key = (key or context.solver).lower()
-
-    solvers_mapping = {}
-    for solver in solvers:
-        solvers_mapping[solver.name.lower()] = solver.backend
-
-    if key in solvers_mapping:
-        return solvers_mapping[key]
-    else:
-        raise CondaValueError(
-            f"You have chosen a non-default solver backend ({key}) "
-            f"but it was not recognized. Choose one of: "
-            f"{', '.join(solvers_mapping.keys())}"
-        )
+    from conda.plugins import solvers
+    # TODO add deprecation warning
```

Comment:
^ Came here to say this :D

Reply:
Done in https://github.com/conda/conda/pull/12093/commits/939991a0b2ec294d9403d30e1971d37111232f4d.